src/core/client/admin/components/UserStatus/BanModal.css

```diff
@@ -1,3 +1,10 @@
 .card {
   max-width: 500px;
 }
+
+.textArea {
+  width: 100%;
+  box-sizing: border-box;
+  height: calc(12 * var(--mini-unit));
```

Comment:
Would it be possible to use a min-height here instead to allow for better responsive scaling?